modules/social_features/social_activity/modules/social_activity_filter/src/Form/FilterSettingsForm.php

```diff
@@ -0,0 +1,174 @@
+<?php
+
+namespace Drupal\social_activity_filter\Form;
+
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\DependencyInjection\ContainerInjectionInterface;
+use Drupal\Core\Entity\EntityFieldManagerInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Extension\ModuleHandlerInterface;
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Class FilterSettingsForm.
+ *
+ * @package Drupal\unpd_cop\Form
```

Comment:
@volodymyr-sydor-lemberg please remove all references to UNPD or COP or from this PR.

Reply:
done